### PR TITLE
Feature/update monthly challenge page to Hacktoberfest

### LINKED ISSUE #991: Update Monthly Challenge — Oct 2023
October monthly challenge is "Hacktoberfest" and we need to change the monthly challenge page of the site.

This year, we need to update the content on the page accordingly, considering:
- Hacktoberfest is no longer giving free swag for participants.
- New repositories list.
`

### PR DESCRIPTION
**Pull request by adiati98**
## Linked Issue

Closes #991 

## Description

This PR adds Hacktoberfest challenge page to the website.

Changes done here:

- Added challenge page in `oct-2023.jsx`.
- Updated `getChallenges.ts`.
- Set `current: true` to Hacktoberfest challenge in `index.tsx`.
- Removed `current: true` from Preptember challenge in `index.tsx`.
- Added blog posts for Hacktoberfest and Preptember challenges in `index.tsx`.
- Added challenge is complete alert to Preptember challenge.

## Live Preview

https://deploy-preview-994--virtual-coffee-io.netlify.app/monthlychallenges/oct-2023

## Code of Conduct

> By submitting this pull request, you agree to follow our [Code of Conduct](https://virtualcoffee.io/code-of-conduct/)

### CODE REVIEW

File: app/routes/__frontend/monthlychallenges/oct-2023.jsx [hunk 1]
```diff
@@ -0,0 +1,483 @@
...
+				spending time in the <code>#hacktoberfest-co-working-room</code>,
+				talking to your mentor, or answering questions as a maintainer?
+			</p>
+			<p>
+				Every Friday of the month, we will drop a check-in for you to share your
+				progress, ask questions, and find help and support.
+			</p>
+			<p>
+				We will drop the check-ins in the <code>#hacktoberfest</code> channel on
+				Slack.
```
Comment (adiati98) [inline app/routes/__frontend/monthlychallenges/oct-2023.jsx:125]: I changed the title to be "Weekly async check-ins" and adjust the wordings as per discussed on Slack.

File: app/routes/__frontend/monthlychallenges/oct-2023.jsx [hunk 2]
```diff
@@ -0,0 +1,483 @@
...
+			<p>
+				Share your progress in the <code>#hacktoberfest</code> channel in Slack
+				so we can celebrate with you! And if you need support, we'll be there to
+				help you, too.
+			</p>
+			<p>
+				While no other platform is imposed, sharing on social media for more
+				reach can be a good idea, but only if you are comfortable doing so. You
+				can share on X (formerly Twitter) using — or not — the hashtag{' '}
+				<code>#VCHI</code>, a personal blog, a post on{' '}
```
Comment (adiati98) [inline app/routes/__frontend/monthlychallenges/oct-2023.jsx:138]: New hashtag suggested from `#VCMonthlyChallenge` to `#VCHI` for this challenge.

File: app/routes/__frontend/monthlychallenges/oct-2023.jsx [hunk 3]
```diff
@@ -0,0 +1,483 @@
...
+			<p>And remember, we're always here to help! 💙</p>
+
+			<h3>Completing the Challenge!</h3>
+			<p>
+				To complete the challenge, you need to complete four meaningful
+				contributions during October. It can be creating a PR, writing a good
+				issue, creating content such as a blog post or video about contributing
+				to a project, helping to answer questions or triaging issues, meeting
+				with your mentee, and so on. Share your progress in the{' '}
+				<code>#hacktoberfest</code> channel on Slack.
```
Comment (adiati98) [inline app/routes/__frontend/monthlychallenges/oct-2023.jsx:165]: As per dicussion on Slack, I changed the wordings here to suit the "four meaningful contributions during October".
Previously:

> Any kind of participation in Hacktoberfest counts as you complete the VC October challenge.

**Review by adiati98 — COMMENTED** [review]
@danieltott @BekahHW I commented the new changes here.
Please let me know if I need to fix anything.
Thank you! 😊

**Review by danieltott — APPROVED** [review]
Thanks @adiati98 !!

**Pull request merged**